### ISSUE TITLE
Training detail of the OryxViT

### ISSUE THREAD

**Issue by royzhang12** [Oryx-mllm/Oryx#19]
Hi @liuzuyan, thanks for providing such a great work. Just a small question about the OryxViT.  As in the paper not much detail of training the OryxViT is described. I am wondering how you train the VIT?

- Is the model initialized from Siglip?
-  Do you use a small llm like 3B or 1.5 model and a simple projector(like the mlp used in llava?) to train the OryxViT by minimizing the cross-entropy of the text tokens?
-  What is the max pixels used in training the OryxViT?
-  What data do you used to train the OryxViT?

**Comment by liuzuyan** [comment]
Hi, thanks for your interest in our work!
1) Yes, the weights of OryxViT are initialized from SigLIP-400M.
2) Yes, we use the Qwen-2-0.5B as a language interface, using a similar architecture as LLaVA to train OryxViT with cross-entropy loss.
3) The training procedure follows the native-resolution setting in the main paper.
4) We collected a total of 400M pre-training data, focusing mainly on image captioning and image OCR tasks. For image captioning, we used the CapsFusion datasets, and for OCR tasks, we employed synthesized OCR data pairs with OCR models.
